gazebo_plugins/src/gazebo_ros_wheel_slip.cpp

```diff
@@ -50,6 +50,23 @@ void GazeboRosWheelSlip::Load(gazebo::physics::ModelPtr _model, sdf::ElementPtr
   // Initialize the WheelSlipPlugin first so its values are preferred unless the ros
   // parameters are overridden by a launch file.
   WheelSlipPlugin::Load(_model, _sdf);
+  double slip_lateral_default = -1.0;
+  double slip_longitudinal_default = -1.0;
```

Comment:
IIUC, if these are not set in SDF, then the parent plugin defaults to zero values.
```suggestion
  double slip_lateral_default = 0.0;
  double slip_longitudinal_default = 0.0;
```